src/pages/terms-of-use.md

```diff
@@ -0,0 +1,203 @@
+---
+template: legal-page
+path: /terms-of-use
+title: Keep Network Terms of Use
+date: "Last Modified: March 30, 2020"
+---
+## Acceptance of the Terms of Use
+
+These terms of use are entered into by and between You and Keep SECZ, a Cayman Islands special economic zone company ("**Company**", "**we**" or "**us**"). The following terms and conditions, together with any documents they expressly incorporate by reference (collectively, these "**Terms of Use**"), govern your access to and use of the network located at [www.keep.network](/) as well as any content, functionality, smart contracts, distributed applications ("Dapps") and services offered on or through the Keep network (collectively, the "**Network**"). The Network is currently administered by the Company using open source protocols; however, we intend over time for the Network and the network protocol to become decentralized and governed entirely by the community of its users in whatever manner they determine.
```

Comment:
`www.keep.network` here should be `https://keep.network`. Probably need to comm that to Josh as well. Also I would leave this as-is rather than linking it to `/`: it being a legal doc, I think it needs to specifically refer to the canonical root of our real site.